### PR TITLE
Detect Ubuntu flavors

### PR DESCRIPTION
**Pull request by arisinfenix**
## Description
Added detection of ubuntu flavors. Solves #1505

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -1087,6 +1087,22 @@ get_distro() {
...
+                done
+
+                case $flavor in
+                    "plasma")  distro=${distro/Ubuntu/Kubuntu} ;;
+                    "mate")    distro=${distro/Ubuntu/Ubuntu MATE} ;;
+                    "xubuntu") distro=${distro/Ubuntu/Xubuntu} ;;
+                    "Lubuntu") distro=${distro/Ubuntu/Lubuntu} ;;
+                    "budgie")  distro=${distro/Ubuntu/Ubuntu Budgie} ;;
+                    "ubuntustudio") distro=${distro/Ubuntu/Ubuntu Studio} ;;
+                esac
```
Comment (dylanaraps) [inline neofetch:1104]: Does something like this not work?

```
case $XDG_CONFIG_DIRS in
    *plasma*) ;;
    ...
esac
```

Comment (arisinfenix) [reply neofetch:1104]: Ah, yes that looks nicer and should also work. I will try it, thanks.

**Comment by arisinfenix** [comment]
Just force pushed the changes and tested it with the flavors live CDs. Works fine.

**Comment by arisinfenix** [comment]
Ah wait, I accidently removed the $distro == Ubuntu test.

**Comment by arisinfenix** [comment]
Added back the test now.

**Comment by dylanaraps** [comment]
Thanks!

**Pull request merged**